### PR TITLE
Create officers page

### LINKED ISSUE #7: Create Officers Page

### PR DESCRIPTION
**Pull request by encadyma**

### CODE REVIEW

File: page-officers.hbs [hunk 1]
```diff
@@ -0,0 +1,427 @@
+{{!--
```
Comment (PotatoParser) [inline page-officers.hbs:1]: Would be nice if you could use a JSON format and then iterate through that to display some of the info, but it looks like Ghost is not capable of that. The manual HTML galore :(

File: page-officers.hbs [hunk 2]
```diff
@@ -0,0 +1,427 @@
...
+                  <p>President</p>
+                </div>
+                <div class="ieee-officer-card">
+                  <img src="{{asset 'images/default-avatar-square-normal.jpg'}}" alt="Neha Hudait"/>
+                  <h3>Neha Hudait</h3>
+                  <p>External VP</p>
+                  <div class="ieee-officer-socials">
+                    <a class="icon-linkedin" href="#"></a>
+                    <a class="icon-instagram" href="#"></a>
+                    <a class="icon-twitter" href="#"></a>
```
Comment (PotatoParser) [inline page-officers.hbs]: I think some visual fixes should be made to these icons. They're too blue in hue and are a little hard to see (a little small).

File: src/sass/components/officers/_officers.scss [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  margin-bottom: 40px;
+}
+
+.ieee-officer-card {
+  background-color: #D0D0E0;
+  border-radius: 2px;
+  padding: 20px;
```
Comment (PotatoParser) [inline src/sass/components/officers/_officers.scss]: Some of the officers have long names which causes the boxes to be a little wider than normal. I think they should be conformed to a set size (looks nicer).

File: src/sass/components/officers/_officers.scss [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+  margin: 10px;
+  text-align: center;
+  
+  img {
+    height: 150px;
```
Comment (PotatoParser) [inline src/sass/components/officers/_officers.scss:47]: You might want to set an `overflow: none;` to allow image cropping, unless we're manually cropping all of the images ;-;

**Review by PotatoParser — CHANGES_REQUESTED** [review]
Overall looks very nice! Though I am kinda sad that everything needs to be put into HTML manually. Also, I think that the names should be ordered alphabetically (either by first name or by last name). Of course, this still means that the directors would come first.

File: page-officers.hbs (hunk 1 above)
Comment (encadyma) [reply page-officers.hbs:1]: That would be nice, but I think leaving it hard-coded as is would be a good exercise for those who maintain the theme next.

File: page-officers.hbs (hunk 2 above)
Comment (encadyma) [reply page-officers.hbs]: Any suggestions on what colors they should be? Would darker icons (closer to black/blue) be a better fit?

File: src/sass/components/officers/_officers.scss (hunk 4 above)
Comment (encadyma) [reply src/sass/components/officers/_officers.scss:47]: Forgot about this, thanks for the suggestions

File: page-officers.hbs (hunk 2 above)
Comment (PotatoParser) [reply page-officers.hbs]: Ohhhhh, that sounds really nice, maybe darken the IEEE blue `#00629B`?

**Comment by encadyma** [comment]
Thanks for the suggestions @PotatoParser , got these in a fix. Can I get another look-through?

**Comment by PotatoParser** [comment]
@encadyma Should we add all the images & social links before merging?

**Comment by encadyma** [comment]
I can add them in for Spring 2020

**Review by PotatoParser — APPROVED** [review]
Sounds like a good idea!

**Pull request merged**